azurelinuxagent/common/utils/networkutil.py

```diff
@@ -93,3 +96,49 @@ def __str__(self):
         if len(self.ipv6) > 0:
             entries.append('"ipv6": {0}'.format(self._json_array(self.ipv6)))
         return "{{ {0} }}".format(", ".join(entries))
+
+
+class AddFirewallRules(object):
+
+    @staticmethod
+    def _add_wait(wait, command):
+        """
+        If 'wait' is True, adds the wait option (-w) to the given iptables command line
+        """
+        if wait:
+            command.insert(1, "-w")
+        return command
+
+    @staticmethod
+    def get_iptables_accept_command(wait, command, destination, owner_uid):
```

Comment:
These 2 commands are all we need to setup the rules on system reboot so just created a new Util just for them